packages/adapters/src/process-instance-adapter.ts

```diff
@@ -49,12 +49,20 @@ IComponent {
 
     getRunnerCmd(config: SequenceConfig) {
         if (config.entrypointPath.endsWith(".py")) {
-            this.logger.trace(gotPython);
+            const engines = Object.keys(config.engines);
 
-            return [
-                "python3",
-                path.resolve(__dirname, "../../../python/runner/runner.py")
-            ];
+            if (engines.length > 1) {
+                throw new Error("Incorrect config pased to SequenceConfig," +
+                "'engines' field can't contain more than one element");
+            }
+
+            if (engines[0] === "python3") {
```

Comment:
```if ("python3" in config.engines) {````
or 
```if (~engines.indexof("python3") {```
of 
```if (engines.includes("pyhon3") {```

coz, "python3" could be on any place in keys if there is more than 1 required engine specified.

Reply:
if (engines.length > 1)  solve this assumption for now, but I've changed as you requested, it's more safe solution for future :)